### PR TITLE
Shopping cart problem -medium

### PR DESCRIPTION
**Pull request by AcidBurn18**

### CODE REVIEW

File: Medium/1. Shopping Cart Problem/solutions/AcidBurn18_AnshAgrawal/Solution [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    f=True
+    while(f):
+        item=input("Items Wants to purchase: ")
+        if item in num:
+            
+            quant=int(input("Quantitiy : "))
+            purchased.append(item)
+            quantity.append(quant)
+            print("Want to buy more Y/N")
+            more=input("Enter ypur choice:- ")
```
Comment (girlscript-blr) [inline Medium/1. Shopping Cart Problem/solutions/AcidBurn18_AnshAgrawal/Solution]: `your` is misspelled.

**Review by girlscript-blr — CHANGES_REQUESTED** [review]
There seems to be a bug in your code, refer to the screenshot attached:
Screenshot 2020-07-11 at 2 19 11 PM

File: Medium/1. Shopping Cart Problem/solutions/AcidBurn18_AnshAgrawal/Solution (hunk 1 above)
Comment (AcidBurn18) [reply Medium/1. Shopping Cart Problem/solutions/AcidBurn18_AnshAgrawal/Solution]: Done

File: Medium/1. Shopping Cart Problem/solutions/AcidBurn18_AnshAgrawal/Solution [hunk 2]
```diff
@@ -18,7 +18,7 @@ t=True
 while(t):
     purchased=[]
     quantity=[]
-    name=input("Entee the Customer Name: ")
+    name=input("Entee=r the Customer Name: ")
```
Comment (girlscript-blr) [inline Medium/1. Shopping Cart Problem/solutions/AcidBurn18_AnshAgrawal/Solution]: spelling mistake.

**Review by girlscript-blr — CHANGES_REQUESTED** [review]
is this https://github.com/girlscript-blr/code-with-girlscript-bangalore/pull/24#pullrequestreview-446789443 change request resolved? I am still getting the same error:

`Traceback (most recent call last):
  File "/Volumes/Macintosh HD 2/MyCodes/ShoppingCartProblemPython/hello.py", line 62, in <module>
    d=shoplist[mapping[purchased[i]]]
KeyError: 'Graphic card'`

**Review by girlscript-blr — APPROVED** [review]

**Comment by girlscript-blr** [comment]
Greetings from Code With Girlscript Bangalore😃

Dear participant, we have gone through your pull requests made in `MEDIUM` sections of the problem statement.

Congratulations🎊 🎉  on getting your pull request successfully merged in the `Shopping Cart Problem - Medium` section in `code-with-girlscript-bangalore` repository. We will be distributing your valuable badges🥇🥈🥉🎖️ corresponding to the pull requests that are merged.

Learning is a never-ending process. 📈 Check out our [Advanced](https://github.com/girlscript-blr/code-with-girlscript-bangalore/tree/master/Hard/1.%20Shopping%20Cart%20Problem) level problem and try your hand at it to earn **THE AFICIONADO** 🥇 and **GEEK OF THE GEEKS** 🏆 badges. Thanks for your contribution! Happy coding😃

**Pull request merged**